commando/tests/test_commando.py

```diff
@@ -293,11 +284,10 @@ def _foobar_blip_blop():
 
 @trap_exit_fail
 def test_nested_command_subcommands():
-    with nested(patch.object(NestedCommandLine, '_main'),
-                patch.object(NestedCommandLine, '_sub'),
-                patch.object(NestedCommandLine, '_foobar_bla'),
-                patch.object(NestedCommandLine, '_foobar_blip_blop')) \
-                as (_main, _sub, _foobar_bla, _foobar_blip_blop):
+    with patch.object(NestedCommandLine, '_main') as _main, \
+         patch.object(NestedCommandLine, '_sub') as _sub, \
+         patch.object(NestedCommandLine, '_foobar_bla') as _foobar_bla, \
+         patch.object(NestedCommandLine, '_foobar_blip_blop') as _foobar_blip_blop:
```

Comment:
I typically avoid line continuations like this, but I don't think there's another way to accomplish what needs to be done here across multiple versions of Python without adding new dependencies.